RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -142,8 +144,8 @@ extension ProfileStore {
 	/// updates `headerList` (Keychain),  `activeProfileID` (UserDefaults)
 	/// and saves the snapshot of the profile into Keychain.
 	/// - Parameter profile: Imported Profile to use and save.
-	public func importProfileSnapshot(_ snapshot: ProfileSnapshot) throws {
-		try importProfile(Profile(snapshot: snapshot))
+	public func importProfileSnapshot(_ snapshot: Profile) throws {
```

Comment:
this and many other methods which use one level of indirection between `Profile` and `ProfileSnapshot` is no longer needed, but, no need to change, since hopefully in next phase we remove `ProfileStore`... (or greatly refactor it) - moving logic into Sargon...